driver/mounter.go

```diff
@@ -208,6 +214,50 @@ func (m *mounter) Unmount(target string) error {
 	return mount.CleanupMountPoint(target, m.kMounter, true)
 }
 
+func (m *mounter) IsRunning(source string, cmdContext execContext) bool {
+	cmd := fmt.Sprintf("ls -l %s | awk '{print $NF}'", source)
```

Comment:
There's a lot of command execution involved which isn't ideal. I think we can get around it.

Starting here, I believe we can use https://pkg.go.dev/path/filepath#EvalSymlinks.